src/ui/inspector/InspectorStylesSpec.js

```diff
@@ -83,7 +83,7 @@ describe("the inspector", () => {
         });
     });
 
-    xit("should allow a saved style to be applied", () => {
+    it("should allow a saved style to be applied", () => {
```

Comment:
This test is failing with an error `Error: Attempted to mutate immutable object test~image thrown`

Reply:
This is probably because there is no `provider` for that object. You will need to create a mock provider for it.